app/models/spree/webinar_registration.rb

```diff
@@ -20,12 +20,18 @@ def sync_with_gotomeeting
     data = SpreeGotomeeting.client.post(
       "/G2W/rest/v2/organizers/%{organizer_key}/webinars/#{product.webinar_key}/registrants",
       json: params
-    ).parse
+    )
+
+    unless data.status.success?
```

Comment:
A couple things:

1. Conditionals with a single statement in the body should be done inline. So `raise('...') unless data.status.success?`
2. I'd include the response data in the exception message, so that we at least have some idea of what happened. Maybe `"Unexpected response from GoToMeeting API: #{data}"`